examples/with-purgecss/tailwind.js

```diff
@@ -0,0 +1,744 @@
+/*
+
+Tailwind - The Utility-First CSS Framework
+
+A project by Adam Wathan (@adamwathan), Jonathan Reinink (@reinink),
+David Hemphill (@davidhemphill) and Steve Schoger (@steveschoger).
+
+Welcome to the Tailwind config file. This is where you can customize
+Tailwind specifically for your project. Don't be intimidated by the
+length of this file. It's really just a big JavaScript object and
```

Comment:
GitHub diff sure got intimidated by the length of this file :rofl:

Reply:
@qm3ster indeed, it is better to include it through package.json and remove tailwind.js from project's repo files.

Reply:
@husayt I based this example on the tailwindcss one. The tailwind.js is the config file. https://tailwindcss.com/docs/installation#2-create-a-tailwind-config-file

Reply:
I see. Then I don't see a problem. Thanks for useful example.